### ISSUE TITLE
separate things by repository

### ISSUE THREAD

**Issue by gutierri** [Laverna/laverna#507]
It seems more organized by separate repositories, web app, desktop app and mobile app.

And maybe let the separate visual, so it becomes easy to view, contribute and build the application.

I think something like:

- Laverna/mobile
- Laverna/desktop
- Laverna/webapp

What do you think?

**Comment by wwebfor** [comment]
Yeah, probably it is a good idea. It's just right now there is almost no difference between the code base of mobile, desktop, and webapp.

**Comment by gutierri** [comment]
Lets separate master code is not an alternative for the next releases? This makes it easier to focus on other applications.

I see that the RC Release has APK.

**Comment by gutierri** [comment]
Or could create forks this repository until they have autonomy being a repository. It therefore so is simple and easy to make changes without affecting the other thing.
